plugins/services/src/js/utils/ServiceStatusIconUtil.ts

```diff
@@ -0,0 +1,59 @@
+import DateUtil from "#SRC/js/utils/DateUtil";
+
+import Pod from "../structs/Pod";
+import Service from "../structs/Service";
+import ServiceTree from "../structs/ServiceTree";
+import DeclinedOffersUtil from "../utils/DeclinedOffersUtil";
+
+interface ServiceQueue {
+  since: string | null;
+}
+
+const isUnableToLaunch = (service: Service | Pod | ServiceTree) => {
+  const queue = service.getQueue() as ServiceQueue | null;
+  const UNABLE_TO_LAUNCH_TIMEOUT = 1000 * 60 * 30;
```

Comment:
we're recreating this constant every time we call `isUnableToLaunch`. let's pull this to the root-level of the file!